server-common/src/main/java/org/apache/gravitino/server/authentication/OAuthConfig.java

```diff
@@ -77,10 +77,10 @@ public interface OAuthConfig {
   // OAuth provider configs
   ConfigEntry<String> PROVIDER =
       new ConfigBuilder(OAUTH_CONFIG_PREFIX + "provider")
-          .doc("The OAuth provider to use (e.g., azure)")
+          .doc("The OAuth provider to use. Supported values: 'default', 'oidc'")
```

Comment:
You would better to explain the config option will be used only for UI.

Reply:
Done